src/k8s-engine/hyperkit.ts

```diff
@@ -246,6 +247,15 @@ export default class HyperkitBackend extends events.EventEmitter implements K8s.
     const finalArgs = defaultArgs.concat(args);
 
     console.log(JSON.stringify([driver].concat(finalArgs)));
+    if (args[0] === 'stop') {
+      // Fire and forget to prevent creation of a long-running promise during shutdown
+      // See https://github.com/rancher-sandbox/rancher-desktop/issues/333
+      execFile(driver, finalArgs);
+
+      return new Promise((resolve) => {
+        resolve();
+      });
```

Comment:
```suggestion
      return Promise.resolve();
```

Of course, since this is already an `async` function, just plain `return;` would do too.